core/model/licensing.py

```diff
@@ -270,6 +270,7 @@ class LicensePool(Base):
     licenses_available: int = Column(Integer, default=0, index=True)
     licenses_reserved: int = Column(Integer, default=0)
     patrons_in_hold_queue = Column(Integer, default=0)
+    should_track_playtime = Column(Boolean, default=False)
```

Comment:
Need to update this to be `nullable=False` as well.

Reply:
No, I just liked my code :(